src/Uno.UWP/UI/Colors.cs

```diff
@@ -1,7 +1,10 @@
-﻿#if !SILVERLIGHT
-using System;
+﻿using System;
 using System.Collections.Generic;
 using System.Text;
+using Windows.UI;
+
+// Required for the WinUI auto upgrade;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5475653)